src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -260,3 +260,28 @@ def set_quotas(self, path, max_bytes=None, max_files=None):
         if max_files is not None:
             self.cfs.setxattr(path, 'ceph.quota.max_files',
                               str(max_files).encode(), 0)
+
+    def get_statfs(self, path):
+        """
+        Get the statfs of the specified path by xattr.
+        :param path: The path of the directory/file.
+        :type path: str
+        :return: Returns a dictionary containing 'used_bytes'
+            and 'used_files'.
+        :rtype: dict
+        """
+        logger.debug("get_statfs by xattr of directory: %s", path)
+
+        try:
+            used_bytes = int(self.cfs.getxattr(path, 'ceph.dir.rbytes'))
+        except cephfs.NoData:
+            used_bytes = 0
+        try:
+            used_files = int(self.cfs.getxattr(path, 'ceph.dir.rfiles'))
+        except cephfs.NoData:
+            used_files = 0
+        try:
+            sub_dirs = int(self.cfs.getxattr(path, 'ceph.dir.rsubdirs'))
+        except cephfs.NoData:
+            sub_dirs = 0
+        return {'used_bytes': used_bytes, 'used_files': (used_files + sub_dirs)}
```

Comment:
better names could be `total_size` and `total_files`

Reply:
files + subdirs is not total files rights, its a mix of files and dirs. Maybe you should segregate them out and return dirs seperately

Reply:
ok

Reply:
```
# ceph daemon /var/run/ceph/ceph-mds.rook-cephfs-b.asok dump inode 1  | grep -A 7 -w rstat
    "rstat": {
        "version": 198,
        "rbytes": 23555211429,
        "rfiles": 36,
        "rsubdirs": 6,
        "rsnaps": 0,
        "rctime": "2023-01-11T14:47:27.131023+0000"
    },
```
Are you sure we will use `total_size` and `total_files`?

Reply:
Dashboard is to display data that is easy to understand right; these are the attributes at FS level. I don't think the naming should be same as the attribute name, for e.g. `total_size` can be easily understood that it's the `rbytes` we're talking about and easy to understand for all. Since it's a matter where people can have varying opinions, it would be good to have some comments from our team. I'll ask a review from FS team.

Reply:
got it. For e.g. `df -h`
```
# df -h
Filesystem      Size  Used Avail Use% Mounted on
udev            252G     0  252G   0% /dev
tmpfs            51G  4.1G   47G   9% /run
/dev/nvme0n1p2  110G   18G   87G  17% /
tmpfs           252G  131M  252G   1% /dev/shm
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           252G     0  252G   0% /sys/fs/cgroup
```
`Used`, is it our answer?

Reply:
if you decide to use "used" then i think used_bytes look better

Reply:
got it, we keep name unchanged, and segregate files and subdirs.